valhalla/userrequests/request_utils.py

```diff
@@ -99,3 +99,22 @@ def get_airmasses_for_request_at_sites(request_dict):
                 data['airmass_limit'] = constraints['max_airmass']
 
     return data
+
+
+def exposure_completion_percentage_from_pond_block(pond_block):
```

Comment:
Any tests for this method?

Reply:
Yeah, the method that calls this method is tested in test_state_changes.py TestStateFromPondBlocks. I didn't modify the method, I just moved the file it was in.